test/utils/Utils.ts

```diff
@@ -29,7 +30,6 @@ export async function loginWithIdentity(serverUrl: string, identity, options?: P
         ...options,
         getLocalStorage,
         createOpts: {
-            request
```

Comment:
I guess tests are failing because of this removal, we managed to use `request` just for the tests.